src/test/Notification Env.postman_environment.json

```diff
@@ -0,0 +1,39 @@
+{
+	"id": "24d980c1-543a-4358-9c56-055e39bf7a82",
+	"name": "Notification Env",
+	"values": [
+		{
+			"key": "token",
+			"value": "",
+			"enabled": true
+		},
+		{
+			"key": "notification_url",
+			"value": "http://localhost:8080",
+			"enabled": true
+		},
+		{
+			"key": "teams_url",
+			"value": "",
+			"enabled": true
+		},
+		{
+			"key": "rocket_url",
+			"value": "",
+			"enabled": true
+		},
+		{
+			"key": "return_url",
+			"value": "http://www.google.com",
+			"enabled": true
+		},
+		{
+			"key": "generated_url",
+			"value": "",
+			"enabled": true
+		}
+	],
+	"_postman_variable_scope": "environment",
+	"_postman_exported_at": "2020-08-12T17:01:13.876Z",
+	"_postman_exported_using": "Postman/7.29.1"
+}
```

Comment:
what is this

Reply:
It is an environment file for postman so you can set variables within the postman tests and set them in one place to reuse in multiple tests